packages/react-components/src/CryptoType.tsx

```diff
@@ -1,14 +1,14 @@
 // Copyright 2017-2021 @polkadot/react-components authors & contributors
 // SPDX-License-Identifier: Apache-2.0
 
-import type { AccountId, AccountIndex, Address } from '@polkadot/types/interfaces';
+import type { AccountIdIsh } from '../../page-accounts/src/util';
```

Comment:
We should not import from here.

- Generally (this is broken in a few places) we don't want to import cross-pages. 
- Since react-components is top-most level, don't want ti import from a page
- Where imports are used, it should be via `@polkadot/app-accounts`

In this case move the type to react-components and app-accounts can import it. (Or even duplicate)

Reply:
✅ I hope I found all of the bad imports